normandy/recipes/tests/test_serializers.py

```diff
@@ -28,6 +31,46 @@ def test_it_works(self, rf):
             'is_approved': recipe.is_approved
         }
 
+    def test_it_validates_arguments(self):
+        schema = json.loads(
+            open('normandy/recipes/static/actions/console-log/package.json').read()
+        )
+
+        ActionFactory(
+            name='console-log',
+            arguments_schema=schema['normandy']['argumentsSchema']
+        )
+
+        # If the action specified cannot be found, raise validation
+        # error indicating the arguments schema could not be loaded
+        serializer = RecipeSerializer(data={
+            'action': 'action-that-doesnt-exist', 'arguments': {}
+        })
+
+        serializer.is_valid()
+        assert serializer.errors['arguments'] == ['Could not find arguments schema.']
+        assert pytest.raises(serializers.ValidationError)
+
+        # If the action can be found, raise validation error
+        # with the arguments error formatted appropriately
+        serializer = RecipeSerializer(data={
+            'action': 'console-log', 'arguments': {'message': ''}
+        })
+
+        serializer.is_valid()
+        assert serializer.errors['arguments'] == {'message': 'This field may not be blank.'}
+        assert pytest.raises(serializers.ValidationError)
+
+        # If the action can be found, and the arguments passed
+        # are accurate, serializer should be valid
+        serializer = RecipeSerializer(data={
```

Comment:
i'm just guessing Brittany is referring to ESLint complaining about reassigning variables that are arguments of a functions.